apps/api/src/lessons/const/lessonItems.const.ts

```diff
@@ -0,0 +1,3 @@
+export const LESSON_ITEM_TYPES = ["text_block", "file", "question"];
+
+export type LessonItemType = (typeof LESSON_ITEM_TYPES)[number];
```

Comment:
**suggestion**: Consider adding `as const` to `LESSON_ITEM_TYPES` to make it a readonly. 
```suggestion
export const LESSON_ITEM_TYPES = ["text_block", "file", "question"] as const;

export type LessonItemType = (typeof LESSON_ITEM_TYPES)[number];
```